styles/_theme.scss

```diff
@@ -14,87 +14,172 @@ See the License for the specific language governing permissions and
+$honeydew: #FFC001 !default;
+$pear: #D7E200 !default;
+$lime: #80D34A !default;
+$green-apple: #00D688 !default;
+$plum: #6414EE !default;
+$dragon-fruit: #B458FC !default;
+$raspberry: #FF4C6C !default;
+$orange: #FF720E !default;
+
+// Vivid Sky Colors
+$vivid-sky1: #0BE1FF !default;
+$vivid-sky2: #75EEFF !default;
+$vivid-sky3: #B5F6FF !default;
+$vivid-sky4: #F4FEFF !default;
+
+// Sea Green Colors
+$sea-green1: #00F4C8 !default;
+$sea-green2: #75FFE6 !default;
+$sea-green3: #B5FFF1 !default;
+$sea-green4: #F4FFFD !default;
+
+// Guppie Green Colors
+$guppie-green1: #0BFF8A !default;
+$guppie-green2: #75FFBD !default;
+$guppie-green3: #B5FFDB !default;
+$guppie-green4: #F4FFFA !default;
+
+// Kiwi Colors
+$kiwi1: #80D34A !default;
+$kiwi2: #B1E490 !default;
+$kiwi3: #D5F0C3 !default;
+$kiwi4: #F9FDF6 !default;
+
+// Bitter Lemon Colors
+$bitter-lemon1: #D4DF00 !default;
+$bitter-lemon2: #F3FF0B !default;
+$bitter-lemon3: #FBFFB5 !default;
+$bitter-lemon4: #FEFFF4 !default;
+
+// Metallic Yellow Colors
+$metallic-yellow1: #FFC90B !default;
+$metallic-yellow2: #FFE175 !default;
+$metallic-yellow3: #FFEFB5 !default;
+$metallic-yellow4: #FFFDF4 !default;
+
+// Spray Tan Colors
+$spraytan1: #FFA14A !default;
+$spraytan2: #FFB775 !default;
+$spraytan3: #FFD8B5 !default;
+$spraytan4: #FFF9F4 !default;
+
+// Tulip Colors
+$tulip1: #FF928A !default;
+$tulip2: #FFBAB5 !default;
+$tulip3: #FFE1DF !default;
+$tulip4: #FFF5F4 !default;
+
+// Pink Elephants Colors
+$pink-elephants1: #FF9FEA !default;
+$pink-elephants2: #FFB5EF !default;
+$pink-elephants3: #FFCAF3 !default;
+$pink-elephants4: #FFF4FD !default;
+
+// Lavender Colors
+$lavender1: #E88AFF !default;
+$lavender2: #F0B5FF !default;
+$lavender3: #F9DFFF !default;
+$lavender4: #FDF4FF !default;
+
+// Lilac Colors
+$lilac1: #BBA7F7 !default;
+$lilac2: #D9CEFB !default;
+$lilac3: #E8E2FC !default;
+$lilac4: #F7F5FE !default;
+
+// Jordy Blue Colors
+$jordy-blue1: #8AABFF !default;
+$jordy-blue2: #CAD9FF !default;
+$jordy-blue3: #DFE8FF !default;
+$jordy-blue4: #F4F7FF !default;
+
+// Lake Colors
+$lake1: #4ABEFF !default;
+$lake2: #9FDDFF !default;
+$lake3: #CAECFF !default;
+$lake4: #F4FBFF !default;
+
+// Mustard Colors
+$mustard1: #FFE50B !default;
+$mustard2: #FFF075 !default;
+$mustard3: #FFF7B5 !default;
+$mustard4: #FFFEF4 !default;
+
+// Code Block Backgrounds Colors
+$secondary: #F4FBFF !default;
+$tertiary: #FFF9F4 !default;
+$quternary: #F9FDF6 !default;
+$quinary: #FFF4FD !default;
+
+// Code Block Light Colors
+$background: #F7F8FB !default;
+$text: #11192E !default;
+$highlight: #E3E8F4 !default;
+$selector: #0F2FDA !default;
+$function: #7300B9 !default;
+$number: #00507D !default;
+$variable: #00573A !default;
+
+// Code Block Dark Colors
+$background-dark: #11192E !default;
+$text-dark: #F7F8FB !default;
+$highlight-dark: #29334D !default;
+$selector-dark: #9FDDFF !default;
+$function-dark: #FFAF8C !default;
+$number-dark: #0BE1FF !default;
+$variable-dark: #B1E490 !default;
```

Comment:
I think it might be worth prefixing these all with `code-` to make their usage clear?